src/native/corehost/apphost/bundle_marker.cpp

```diff
@@ -9,11 +9,11 @@
 int64_t bundle_marker_t::header_offset()
 {
     // Contains the bundle_placeholder default value at compile time.
-    // If this is a single-file bundle, the first 8 bytes are replaced 
+    // If this is a single-file bundle, the first 8 bytes are replaced
```

Comment:
Can you please do the whitespace change as a followup? It's basically unrelated to the PR - I'll be happy to take it, just as a separate PR.